app/db/schemas/schemaV4.js

```diff
@@ -0,0 +1,26 @@
+import ProgramLanguage from '../migrations/v3/programLanguage';
+import Scorecard from '../migrations/v4/scorecard';
+import schemaHelper from '../../helpers/schema_helper';
+import { schemaNames } from '../../constants/schema_constant';
+
+const changedSchemas = [
+  { label: schemaNames[0], data: Scorecard },
+  { label: schemaNames[13], data: ProgramLanguage },
+];
+
+const schemaV4 = {
+  schema: schemaHelper.getSchemas(changedSchemas),
+  schemaVersion: 4,
+  migration: (oldRealm, newRealm) => {
+    if (oldRealm.schemaVersion < 4) {
+      const oldObjects = oldRealm.objects('Scorecard');
+      const newObjects = newRealm.objects('Scorecard');
+
+      for (let i = 0; i < oldObjects.length; i++) {
+        newObjects[i].milestone = !oldObjects[i].milestone ? '' : oldObjects[i].milestone;
```

Comment:
@limkimsan this migration seems that get nothing as the oldObject(scorecard was empty on this milestone attribute), it should be s.th like migrate the current data following the stage scorecard encountered to set on this new attribute migration